lib/rules/color-named/index.js

```diff
@@ -58,11 +58,23 @@ const rule = function (expectation, options) {
         return
       }
 
+      const functionNodeIndexes = []
+
       valueParser(decl.value).walk(node => {
         const value = node.value,
           type = node.type,
           sourceIndex = node.sourceIndex
 
+        // Find nodes that are inside functions
+        if (type === "function") {
+          node.nodes.forEach((node) => functionNodeIndexes.push(node.sourceIndex))
+        }
+
+        // Return early if node is inside of a function
+        if (functionNodeIndexes.indexOf(sourceIndex) > -1) {
```

Comment:
Can we accomplish the same thing in a more standard way with [blurFunctionArguments](https://github.com/stylelint/stylelint/blob/master/lib/utils/blurFunctionArguments.js)? Have you checked out what is happening in other rules that also ignore function arguments?